### PR TITLE
Match `Sram_CopySave`

### PR DESCRIPTION
**Pull request by engineer124**
Finally

### CODE REVIEW

**Review by isghj5 — APPROVED** [review]
lol

**Review by hensldm — APPROVED** [review]
Nice :)

**Review by sonicdcer — APPROVED** [review]
Outstanding work!

File: src/code/z_sram_NES.c [hunk 1]
```diff
@@ -1737,13 +1735,11 @@ void Sram_CopySave(FileSelectState* fileSelect2, SramContext* sramCtx) {
         // clear buffer
         bzero(sramCtx->saveBuf, SAVE_BUFFER_SIZE);
         // read to buffer
-        SysFlashrom_ReadData(&sramCtx->saveBuf[0], gFlashSaveStartPages[fileSelect->selectedFileIndex * 2],
-                             gFlashSaveNumPages[fileSelect->selectedFileIndex * 2]);
-        if (1) {}
+        if (SysFlashrom_ReadData(&sramCtx->saveBuf[0], gFlashSaveStartPages[fileSelect->selectedFileIndex * 2],
+                                 gFlashSaveNumPages[fileSelect->selectedFileIndex * 2])) {}
```
Comment (AngheloAlf) [inline src/code/z_sram_NES.c:1739]: oh wow...

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**